allennlp/service/servable/models/simple_tagger.py

```diff
@@ -0,0 +1,61 @@
+import codecs
+import os
+from tempfile import gettempdir
+
+from allennlp.common import Params
+from allennlp.data.dataset_readers.sequence_tagging import SequenceTaggingDatasetReader
+from allennlp.data.fields import TextField
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+from allennlp.data.tokenizers import WordTokenizer
+from allennlp.data.vocabulary import Vocabulary
+from allennlp.models.simple_tagger import SimpleTagger
+from allennlp.service.servable import Servable, JSONDict
+
+# Simple Tagger Model
+class SimpleTaggerServable(Servable):
```

Comment:
that is, it seems likely to me that (e.g.) the web demo of one of these models will want to return model-specific information beyond just the results of `model.forward`, which means we need model-specific wrappers around `Model` that know how to provide that